packages/renderer/src/lib/image/ImageInfoUI.ts

```diff
@@ -31,5 +31,5 @@ export interface ImageInfoUI {
   // no tag, we encode <none>
   base64RepoTag: string;
   selected: boolean;
-  inUse: boolean;
+  status: string;
```

Comment:
should the status be 'USED' | 'UNUSED' rather than string ?

Reply:
Good idea (we should probably do this with other types too).

Updated, and force-pushed to rebase.

Reply:
I don't see the type change, did I miss something ?

Reply:
I think first 'force-push' had the right fix but then the second one replaced the fix

fist one: https://github.com/containers/podman-desktop/commit/92b339f5635ee540646289f3dbacbd4db9d6b4b8

Reply:
Doh, must have lost it in the rebase. Most of the files now have merge conflicts too, I'll do another rebase and make sure I pick this up.

Reply:
Rebased and force-pushed, ready for (re)review.